### PR TITLE
[Browser] don't pass redundant args to wasm

### LINKED ISSUE #41471: CoreCLR tests on WASM: System.FormatException: Input string was not in a correct format
### Description

CoreCLR runtime tests failed with this syndrome:
```
Exception:System.FormatException: Input string was not in a correct format.
           at System.Number.ThrowOverflowOrFormatException(ParsingStatus status, TypeCode type)
           at System.Number.ParseInt32(ReadOnlySpan`1 value, NumberStyles styles, NumberFormatInfo info)
           at System.Int32.Parse(String s)
           at Benchstone.BenchF.Adams.Main(String[] argv)
```
Affected tests are:
- JIT/Performance/CodeQuality/Benchstones/BenchF/Adams/Adams/Adams.sh
- JIT/Performance/CodeQuality/BenchmarksGame/binarytrees/binarytrees-2/binarytrees-2.s
- JIT/Performance/CodeQuality/BenchmarksGame/fasta/fasta-1/fasta-1.sh
- JIT/Performance/CodeQuality/BenchmarksGame/mandelbrot/mandelbrot-7/mandelbrot-7.sh
- JIT/Performance/CodeQuality/BenchmarksGame/pidigits/pidigits-3/pidigits-3.sh
- JIT/Performance/CodeQuality/V8/Richards/Richards/Richards.s
- JIT/Performance/CodeQuality/BenchmarksGame/fannkuch-redux/fannkuch-redux-2/fannkuch-redux-2.sh
- JIT/Performance/CodeQuality/BenchmarksGame/fannkuch-redux/fannkuch-redux-5/fannkuch-redux-5.sh
- JIT/Performance/CodeQuality/SIMD/SeekUnroll/SeekUnroll/SeekUnroll.sh
- JIT/Performance/CodeQuality/BenchmarksGame/n-body/n-body-3/n-body-3.sh
- JIT/Performance/CodeQuality/V8/Crypto/Crypto/Crypto.sh
- JIT/Performance/CodeQuality/BenchmarksGame/mandelbrot/mandelbrot-2/mandelbrot-2.sh
- JIT/Performance/CodeQuality/BenchmarksGame/spectralnorm/spectralnorm-3/spectralnorm-3.sh
- JIT/Performance/CodeQuality/BenchmarksGame/fasta/fasta-2/fasta-2.sh
- JIT/Performance/CodeQuality/BenchmarksGame/spectralnorm/spectralnorm-1/spectralnorm-1.sh
- JIT/Performance/CodeQuality/SciMark/SciMark/SciMark.sh
- JIT/Performance/CodeQuality/BenchmarksGame/binarytrees/binarytrees-5/binarytrees-5.sh
- JIT/opt/OSR/example/example.sh

### Reproduction Steps

1. Get dotnet/runtime repo
2. Remove the test(s) that you would like to reproduce from `<repo_root>/src/coreclr/tests/issues.targets`
2. Build WASM: `./build.sh --os Browser --arch wasm --subset mono+libs -c Release`
3. Build runtime tests: `cd src/coreclr/ && ./build-test.sh -skipstressdependencies -excludemonofailures os Browser wasm Release`
4. To run all runtime tests, at the end of the runtime tests build log, there is "Example runtest.sh command". Use that command plus `wasm`. For example, for me it was
`./tests/runtest.sh --coreOverlayDir=/Users/fanyang/Documents/work/dotnet_naricc/runtime/artifacts/tests/coreclr/Browser.wasm.Release/Tests/Core_Root --testNativeBinDir=/Users/fanyang/Documents/work/dotnet_naricc/runtime/artifacts/obj/coreclr/Browser.wasm.Release/tests --testRootDir=/Users/fanyang/Documents/work/dotnet_naricc/runtime/artifacts/tests/coreclr/Browser.wasm.Release --copyNativeTestBin Release wasm`
5. To run a single test, you could do
```
set CORE_ROOT=<repo_root>/artifacts/tests/coreclr/Browser.wasm.Release/Tests/Core_Root     
<repo_root>/artifacts/tests/coreclr/Browser.wasm.Release/JIT/Methodical/switch/switch5/switch5.sh
```

### Regression?

No. WASM is a new product.

### Other information

All of the tests mentioned in this issue were failing with the same syndrome. If the root cause of them are different, feel free to split this issue into multiple ones.

### PR DESCRIPTION
**Pull request by EgorBo**
Fixes https://github.com/dotnet/runtime/issues/41471

tested coreclr tests, libs tests and wasm samples - this argument is not needed anywhere

e.g. args we use to run coreclr tests: `Arguments: --run,example.dll example` the last one appears as `args[0]` in Main (some tests don't expect that).

### CODE REVIEW

File: tools-local/tasks/mobile.tasks/WasmAppBuilder/WasmAppBuilder.cs [hunk 1]
```diff
@@ -219,7 +219,7 @@ public override bool Execute ()
 
         using (var sw = File.CreateText(Path.Join(AppDir, "run-v8.sh")))
         {
-            sw.WriteLine("v8 --expose_wasm runtime.js -- --run " + Path.GetFileName(MainAssembly) + " $*");
+            sw.WriteLine("v8 --expose_wasm runtime.js -- --run " + Path.GetFileName(MainAssembly));
         }
```
Comment (vargaz) [inline tools-local/tasks/mobile.tasks/WasmAppBuilder/WasmAppBuilder.cs:223]: The purpose of this is to allow running run-v8.sh with arguments.

Comment (EgorBo) [reply tools-local/tasks/mobile.tasks/WasmAppBuilder/WasmAppBuilder.cs:223]: @vargaz ah, I see

Comment (EgorBo) [reply tools-local/tasks/mobile.tasks/WasmAppBuilder/WasmAppBuilder.cs]: fixed.

**Comment by naricc** [comment]
It looks like some of the WASM tests are failing for an unrelated reason, specifically this one: https://github.com/dotnet/runtime/issues/41520

I'm not sure how those tests got enabled though?

**Review by naricc — APPROVED** [review]
I think some of the tests need more argument handling to work correctly, but this change should let us enable more of them.

**Comment by SamMonoRT** [comment]
Try enabling the failing tests from https://github.com/dotnet/runtime/issues/41471 to ensure they pass

**Review by akoeplinger — APPROVED** [review]

File: src/tests/issues.targets [hunk 2]
```diff
@@ -3380,20 +3350,26 @@
...
-            <Issue>https://github.com/dotnet/runtime/issues/41471</Issue>
-        </ExcludeList>
         <ExcludeList Include = "$(XunitTestBinBase)/JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-5/**">
             <Issue>https://github.com/dotnet/runtime/issues/41472</Issue>
         </ExcludeList>
         <ExcludeList Include = "$(XunitTestBinBase)/JIT/Performance/CodeQuality/SIMD/RayTracer/RayTracer/**">
             <Issue>needs triage</Issue>
         </ExcludeList>
+        <ExcludeList Include = "$(XunitTestBinBase)/GC/Scenarios/DoublinkList/dlbigleakthd/*">
+            <Issue>https://github.com/dotnet/runtime/issues/41471</Issue>
```
Comment (lewing) [inline src/tests/issues.targets:3363]: Is this still the right issue for these?

Comment (EgorBo) [reply src/tests/issues.targets:3363]: these fail after my change, apparently need some input args but this PR enables more tests than disables

**Pull request merged**